src/SignalR/clients/java/signalr/signalr.iml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think we'd prefer not to check this in right now. It's an IntelliJ project I presume?